client/src/components/Form/Form.jsx

```diff
@@ -31,15 +36,30 @@ const Form = () => {
       },
       credentials: "include"
     })
-      .then((response) => response.json())
+      .then((response) => {
+        if (response.ok){
+          response.json()
+        }
+        else if (response.status == 404){
+          throw Error('Patient not found');
+        } 
+        else if (response.status == 422){
+          throw Error ('Multiple Matches. Please enter more information.');
```

Comment:
We might want to align with the [IIS Issuer Reference Designs for the text here](https://projects.invisionapp.com/share/5M11EMPTJH7N#/screens/455685499?browse). There is text there for the multi-match error. I don't see any specific instruction for Patient Not Found, but what you have for that error message seems clear to me.

Reply:
Ah didn't see that - good catch. Aligned the text and slight modification to Patient Not Found